docs/references/fluxyaml-config-files.md

```diff
@@ -1,229 +1,268 @@
-# Manifest factorization through `.flux.yaml` configuration files
+# Manifest generation through `.flux.yaml` configuration files
 
-## Enabling search of `.flux.yaml` files
+This feature lets you generate Kubernetes manifests with a program,
+instead of having to include them in your git repo as YAML files. For
```

Comment:
```suggestion
instead of having to include them in your Git repository as YAML files. For
```

I am not sure we are consistent in terminology across the docs. If we use _git_ and _repo_ everywhere else I am fine with it. `Consistency > formal pickiness`

Reply:
We use "git repo" and "git repository" here and there. Of those I think I prefer "git repository". Using a capital gives it a bit too much emphasis I reckon.

Reply:
I've used "git repository" consistently, now.